### PR TITLE
Platform reviewers need not assign unassigned draft PRs

### PR DESCRIPTION
**Pull request by ggould-tri**
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/14368)

### CODE REVIEW

**Review by ggould-tri — COMMENTED** [review]
+@jwnimmer-tri +(status: single reviewer ok)
People have switched to using draft instead of the "do not review" tag, since that's what it's for.

> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/14368#-:-MMVyK0UCXREVedySUoK:bnx7o3d)* status: LGTM missing from assignee jwnimmer-tri(platform) (waiting on @jwnimmer-tri)

**Comment by jwnimmer-tri** [comment]
I seem to recall this came up before.  So we _should_ clarify the question with some text, but I think the opposite direction.

Here's the thought experiment: if a random first-time contributor decides to contribute a PR, but so far only has an unfinished implementation, what would they do?  They might open a "draft PR" with the WIP and say "Can someone help me out?"   If that happens, should we ignore it indefinitely?  If we use the new filter, we would.

In short, the concepts of "the code is complete" vs "this needs attention" are not direct opposites.

If a developer does not want review, they should mark "do not review".

**Review by ggould-tri — COMMENTED** [review]
Huh -- I had a different use case of draft in mind!  Habits are a prison.

What do you think of the amended text below?

Also -(status: single reviewer ok) since this is clearly not the uncontroversial change I thought.

> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/14368#-:-MMWIHqg8GPoEcCgyvBI:bevjh9)* status: LGTM missing from assignee jwnimmer-tri(platform) (waiting on @jwnimmer-tri)

**Review by jwnimmer-tri — COMMENTED** [review]
Can you describe a case where the old text is ambiguous, but where the new text is prescriptive?  I don't think we've had PRs show up in the report recently that were not created by core developers?

> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/14368#-:-MMWPdKw92BVPUW5T1sp:b-5n5e33)* status: LGTM missing from assignee jwnimmer-tri(platform), needs at least two assigned reviewers

**Comment by ggould-tri** [comment]
I think it's clear that there's not an obviously preferable resolution here, so I'm closing it in favor of just continuing the current workflow.
